packages/@okta/vuepress-site/docs/guides/configure-user-scoped-account-management/main/index.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Configure User-Scoped account management
+meta:
+  - name: description
+    content: The MyAccount API now provides user-scoped endpoints that don’t require admin tokens. End users only need a bearer token to update their email and phone authenticators.
+layout: Guides
+sections:
+- main
+---
+
+This guide explains how to set up end user-driven account management in your org using the MyAccount API.
+
+<!-- > **Note:** This guide covers a narrow set of use cases that call the MyAccount API. Most use cases are covered by the SDKs. See -->
+
+---
+**Learning outcomes**
+
+* Set up your org with the required scopes to call the MyAccount API.
+* Allow your end users to call the API to update their profile and enroll another authenticator after first sign-in.
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+* Access to the latest version of the MyAccount API: `/idp/myaccount`
+* An existing OpenID Connect client app in Okta for testing in Okta
+[Postman client](https://www.getpostman.com/downloads/) to test requests. See [Get Started with the Okta APIs](https://developer.okta.com/code/rest/) for information on setting up Postman.
+* The MyAccount API Postman collection that allows you to test the API calls that are described in this guide. Click **Run in Postman** to add the collection to your Postman workspace.
+
+[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/9cb68745dbf85ae3a871)
+
+> **Note:** To run this Postman collection, you need an end-user access token. Use a password grant type or SDK to get the token. See [Embedded Auth with SDKs React](https://github.com/okta/okta-auth-js/tree/master/test/apps/react-oie) for a sample app that provides an access token.
+
+---
+
+## About User-Scoped account management
+
+The MyAccount API now provides user-scoped endpoints that don’t require admin tokens. End users only need a bearer token to update their profile, or email and phone authenticators. In addition, app developers can call the MyAccount API for active users outside of the authentication context. For example, once a user enrolls in the mandatory email factor and completes authentication, app developers can call the API to enroll the active user with the optional phone authenticator.
+
+## About the Postman collection
+
+It is up to you how you set up users to call the MyAccount API to manage their account. In this guide, we provide sample API calls using a Postman collection to demonstrate them in a language/platform neutral way.
+
+> **Note:** To run the Postman collection, you need an end-user access token. Use a password grant type or [SDK](/docs/guides/auth-js/-/main/) to get the token.
```

Comment:
Use a password grant type? You aren't implying that they use the Resource Owner Password grant type, right? If so, it's just to walk through this use case, right? Because we discourage the use of the ROPassword grant type. Unless I'm not understanding what you are telling them to use....which is very possible. haha